src/frontend/komponenter/Felleskomponenter/LeggTilBarn.tsx

```diff
@@ -43,9 +52,14 @@ export interface IRegistrerBarnSkjema {
 interface IProps {
     barnaMedOpplysninger: Felt<IBarnMedOpplysninger[]>;
     onSuccess?: (barn: IPersonInfo) => void;
+    manuelleBrevmottakere: SkjemaBrevmottaker[] | IRestBrevmottaker[];
```

Comment:
hvordan kan denne være begge typene? Har ikke sett noe på den andre koden som er gjort, så mulig det er veldig logisk 😅

Reply:
Det er fordi det kommer manuelleBrevmottakere også på fagsak i neste oppgave, og den er av typen SkjemaBrevMottaker. Eneste forskjellen på de to interfacene er at IRestBrevmottaker extender SkjemaBrevmottaker med en `id`